jdi-light-bootstrap-tests/src/main/java/io/github/com/pages/BootstrapPage.java

```diff
@@ -749,10 +708,4 @@
     public static ModalOptionalSizes modalOptionalSizes;
     //endregion
 
-    //region Forms
-    @UI("#form-horizontal-label-size")
-    public static FormHorizontalLabelSizing formHorizontalLabelSizing;
-    @UI("#form-column-size")
-    public static FormColumnSizing formColumnSizing;
-    //endregion
 }
```

Comment:
You should leave an empty line at the end of file

Reply:
There is an empty line